extensions/easy-ocr/src/index.tsx

```diff
@@ -45,3 +68,14 @@ export default async function main() {
     });
   }
 }
+
+async function autoDetectedLanguage(text: string) {
+  if (!getPreferenceValues<Preferences>().autodetect_lang) {
+    return;
+  }
+
+  // Detect language
+  return await detect(text, {
+    languageCodeFormat: LanguageCodeFormat.ISO_639_3,
+  });
```

Comment:
```suggestion
  return detect(text, {
    languageCodeFormat: LanguageCodeFormat.ISO_639_3,
  });
```

FYI: https://stackoverflow.com/questions/38708550/difference-between-return-await-promise-and-return-promise

Reply:
Thanks for this!